tests/integration/components/frost-bunsen-detail/selected-tab-name-test.js

```diff
@@ -0,0 +1,61 @@
+import {expect} from 'chai'
```

Comment:
My point is that you render a bunsen detail view with the selectedTabLabel as `Two`. The UI comes up and the second tab is selected. You then programmatically change the selectedTabLabel property to `One`. At this point I'd expect the first tab to become selected. This test isn't testing your use case in particular but bunsen is an open source addon consumed by many people with different use cases. If we are going to add this feature I want to make sure it is versatile in it's usage for whatever use cases come up.